api/v1alpha1/ingressnodefirewallconfig_types.go

```diff
@@ -24,6 +24,8 @@ import (
 // IngressNodeFirewallConfigSpec defines the desired state of IngressNodeFirewallConfig
 type IngressNodeFirewallConfigSpec struct {
 	// Nodes where the daemonset will be running on.
+	// + this field is using map[string]string instead of metav1.LabelSelector because it matches the daemonset NodeSelector template
```

Comment:
Same question here, what's the meaning of ` + `.
`This field is using map[string]string instead of metav1.LabelSelector because it matches the daemonset NodeSelector template. Thus one cannot benefit from additional match criteria that metav1.LabelSelector provides.`

Reply:
same answer just for developer comments no visible in the APIs